README.md

```diff
@@ -29,27 +29,36 @@ The Jupynium server will receive events from Neovim, keep the copy of the buffer
 ### Requirements
 
 - 💻 Linux, macOS and Windows (CMD, PowerShell, WSL2)
-- 🦊 Firefox (Other browsers are not supported due to their limitation with Selenium)
-- 🦎 Mozilla geckodriver (May be automatically installed with Firefox. Check `geckodriver -V`)
 - ✌️ Neovim >= v0.8
+- 🦊 Firefox
+  - Other browsers are not supported due to their limitation with Selenium
```

Comment:
Let's add this reference: https://github.com/kiyoon/jupynium.nvim/issues/49#issuecomment-1443304753